scenario/TransferScenario.ts

```diff
@@ -92,8 +92,8 @@ scenario(
     const baseTransferred = getExpectedBaseBalance(50n * scale, baseIndexScale, baseSupplyIndex);
     const baseOfTransferrer = getExpectedBaseBalance(baseSupplied - toTransfer, baseIndexScale, baseSupplyIndex);
 
-    expect(await comet.balanceOf(albert.address)).to.be.equal(baseOfTransferrer);
-    expect(await comet.balanceOf(betty.address)).to.be.equal(baseTransferred);
+    expectBase(await albert.getCometBaseBalance(), baseOfTransferrer);
+    expectBase(await betty.getCometBaseBalance(), baseTransferred);
```

Comment:
Unrelated but intermittently failing while testing this

Reply:
`comet.balanceOf(actor)` and `actor.getCometBaseBalance()` aren't identical. looks like `getCometBaseBalance` subtracts the borrow balance.

was the original test incorrect?

Reply:
oh, I see `expectBase` does an approximate match, not an exact one